Vagrantfile

```diff
@@ -3,10 +3,11 @@ Vagrant.configure("2") do |config|
     config.vm.box_version = ">= 0.3, < 0.4"
     config.vm.provider "virtualbox" do |vb|
         vb.linked_clone = true
-        vb.memory = 2048
+        vb.memory = 4096
+        vb.cpus = 4
```

Comment:
Consider less here. Not all computers can affort 4 cores. 3 might be more appropriate

Reply:
I think I can start with 2 core